### PR TITLE
#11-1 Create Event_create page (Storage DB)

### PR DESCRIPTION
**Pull request by kokioba95**
## Description / Purpose (Checkpoint)
This PR is a test that is stored in the DB by entering foam at event_create.

I fixed the problem.
<https://drive.google.com/open?id=1Ear__g58irVlaI_tpzWUkf__JRcBz8u2d2DdRYKMiDk> <Back Log # 11>

# # To test this PR
1. Go to the login page (example: localhost: 8086 / login).
2. Input user credentials
3. Click Login
4. my_profile page
5. Click the event_create button
6. Enter all content
7. Press the "post" button
8. (Store in DB & return to my_profile page)

** Command **
#Php artisan make: model Event
#Php artisan make: contoller CreateController
  #php artisan make: migration Create

 **screenshot**
-<Https://drive.google.com/open?id=1UX_at7VJu0OeH5Mb7Eo7ULK3hSw7Ws-W>

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -16,7 +16,14 @@
 });
 Auth::routes();
 Route::get('/my_profile', 'HomeController@index')->name('my_profile');
+Route::get('/event_create', 'HomeController@create')->name('event_create');
+Route::post('/post', 'HomeController@post')->name('post');
 Route::get('/user/{id}/edit', 'UserController@edit');
 Route::get('/user/{id}/update', 'UserController@update');
+Route::get('/user/{id}/set', 'UserController@set');
+Route::get('/user/{id}/supporter_update', 'UserController@supporter_update');
+>>>>>>> Stashed changes
```
Comment (AubreyV) [inline routes/web.php]: Please do not push conflicts.

Comment (kokioba95) [reply routes/web.php]: I have modified.

File: app/relationship.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class relationship extends Model
```
Comment (AubreyV) [inline app/relationship.php]: The first letter of the class name should be capitalized based on Laravel convention. Please capitalize also the first letter of file name.

Comment (kokioba95) [reply app/relationship.php]: I had modified.
I will write code based on the rules from now on.

**Pull request merged**